client/app/scripts/liveblog-edit/directives.js

```diff
@@ -98,6 +98,9 @@ define([
                         isPostsEmpty: function() {
                             return vm.pagesManager.count() < 1 && !vm.isLoading;
                         },
+                        isSinglePost: function() {
+                            return vm.pagesManager.count() === 1 && !vm.isLoading;
```

Comment:
Aren't we mixing different behaviour here ? 
Is it the role of the function `isSinglePost` to return false if the controller is, for any reason, loading, even if there is actually one single post ?